### DISCUSSION TITLE
lpf = flopy.modflow.ModflowLpf(mf, hk , vka , ipakcb）

### DISCUSSION THREAD

**Discussion by wqewqeqsaddadasd** [modflowpy/flopy#1516] [Q&A] [answered]
lpf = flopy.modflow.ModflowLpf(mf, hk , vka , ipakcb）

may i sak what's the meaning of "ipakcb"?What does this parameter mean

very thanks!

**Comment by mbakker7** [comment, marked as answer]
The documentation of the function gives the meaning of all parameters:

ipakcb : int
    A flag that is used to determine if cell-by-cell budget data should be
    saved. If ipakcb is non-zero cell-by-cell budget data will be saved.
    (default is 0)

**Reply by wqewqeqsaddadasd** [reply]
very thanks！I am a beginner, can I ask where to download this function document?

**Reply by wqewqeqsaddadasd** [reply]
i think i find it!thanks again!
